### ISSUE TITLE
Rebuiding the modpe scripts and the engine for modding

### ISSUE THREAD

**Issue by cdgames-tech** [zhuowei/MCPELauncher#1515]
Hey i am is a modder but the engine is outdade and the modpe scripts are bad (for entity) but i'm is a developer i'm cam make a new engine from this source code ???

**Comment by DevGod6969** [comment]
yes you can make bec its pretty much discontinued and i don't think it will be comming back anytime soon

**Comment by JRBros2346** [comment]
Could you please give a link for the apk you developed here? I want to try it

**Comment by JRBros2346** [comment]
Could you please give a link for the apk you developed here? I want to try it
